ETABS_oM/Requests/PierAndSpandrelForceRequest.cs

```diff
@@ -37,7 +37,7 @@ public class PierAndSpandrelForceRequest : IStructuralResultRequest
         /***************************************************/
 
         [Description("Defines which type of results that should be extracted.")]
-        public virtual PierAndSpandrelResultType ResultType { get; set; } = PierAndSpandrelResultType.PierForce;
+        public virtual PierAndSpandrelResultType ResultType { get; set; } = new PierAndSpandrelResultType();
```

Comment:
Suggestion implemented and committed.

Reply:
Has this been added to the `BarResult` file I linked? You can use that to call it rather than the current implementation. I have not checked the test script but we want to have the same workflow for pulling `SpandrelForce` as we do all results.